src/Backend/Modules/MediaLibrary/Domain/MediaItem/MediaItem.php

```diff
@@ -235,21 +258,28 @@ public static function createFromMovieUrl(
     public static function fromDataTransferObject(MediaItemDataTransferObject $mediaItemDataTransferObject): ?MediaItem
     {
         if (!$mediaItemDataTransferObject->hasExistingMediaItem()) {
-            throw new \BadFunctionCallException('This method can not be used to create a new media item');
+            throw new BadFunctionCallException('This method can not be used to create a new media item');
```

Comment:
Exception thrown if a callback refers to an undefined function or if some arguments are missing. 

I think a RuntimeException would be better here

Reply:
But,... some arguments are missing? That is why I throw error.

Reply:
Fixed. Changed to RuntimeException as requested.